benchmarks/README.md

```diff
@@ -81,3 +81,18 @@ These steps help reproduce the failures in TFLite models.
 5. Then compile the input MLIR file with `iree-compile`. The exact flags used
    to compile and run the benchmarks can be found in
    [this CMakeLists.txt file](./TFLite/CMakeLists.txt).
+
+### Run benchmark suites locally
+
+First you need to have `iree-import-tflite` and `requests` in your python environment. Then you can build the target `iree-benchmark-suites` to generate the required files:
```

Comment:
nit: wrapping lines at 80 (or 100) characters would make these files a little easier to view in narrow terminals and web interfaces (and it helps to be consistent with the rest of the file)

same down below, a few lines are pretty long